src/modules/Core/selectors.js

```diff
@@ -50,15 +52,8 @@ export const getCurrencyConverter = (state: State) => {
   return currencyConverter
 }
 
-export const getExchangeRate = (state: State, fromCurrencyCode: string, toCurrencyCode: string) => {
-  const currencyConverter = getCurrencyConverter(state)
-  const exchangeRate = currencyConverter.convertCurrency(fromCurrencyCode, toCurrencyCode, 1)
-  return exchangeRate
-}
-
 export const getFakeExchangeRate = (state: State, fromCurrencyCode: string, toCurrencyCode: string) => {
-  const currencyConverter = getCurrencyConverter(state)
-  const exchangeRate = currencyConverter.convertCurrency(fromCurrencyCode, toCurrencyCode, 1)
+  const exchangeRate = convertCurrency(state, fromCurrencyCode, toCurrencyCode, 1)
   return exchangeRate + Math.random() * 10
```

Comment:
We will only be calling `currencyConverter.convertCurrency` from one area of the GUI